src/window.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+** EPITECH PROJECT, 2022
+** indieStudio [WSL : Ubuntu]
+** File description:
+** window
+*/
+#include "window.hpp"
+#include <raylib.h>
+
+indie::window::window(){
+    const int screenWidth = GetMonitorWidth(0);
+    const int screenHeight = GetMonitorHeight(0);
+
+    InitWindow(screenWidth, screenHeight, "raylib [shapes] example - basic shapes drawing");
+}
+
+
+indie::window::~window(){
+        CloseWindow();
+}
+
+int indie::window::get_key_pressed(){
+    int key = GetKeyPressed();
```

Comment:
I'm gonna need different functions for my event handling but I'll change that myself